docs/ancestry.md

```diff
@@ -2071,6 +2071,27 @@ print(tree.tmrca(0,1))
 
 ```
 
+Because branch lengths are proportional to {math}`N^2` generations, other events (such as mutations or recombinations) will also be proportional to {math}`N^2`. For example, if c is set to 0, the dirac coalescent will behave as if under a Moran model. However if believed census size is inputted (e.g. {math}`N=10000`), a number of segregating sites proportional to {math}`N^2` will be obtained. Leading to unrealistic simulated data.
+
+```{code-cell}
+ts = msprime.sim_ancestry(
+    samples=1, ploidy=2, random_seed=1, population_size=10000,recombination_rate=1e-8,
+    sequence_length=10**4,model=msprime.DiracCoalescent(psi=0.1, c=0))
+mts = msprime.sim_mutations(ts, rate=1e-8)
+print(str(mts.get_num_sites()))
+```
+However if the square root of the population size is inputted, the expected (and desired) ammount of diversity is obtained.
```

Comment:
Continuing from above, how about:

If the desired amount of diversity (as measured by the number of segregating sites) is proportional to {math}`N`, then {math}`N^{1/2}` should be used for the population size under the Dirac-coalescent.